packages/web/app/src/components/target/history/labelize.tsx

```diff
@@ -0,0 +1,10 @@
+import { ReactNode } from 'react';
+import reactStringReplace from 'react-string-replace';
+import { Label } from '@/components/common';
+
+export function labelize(message: string): ReactNode[] {
+  // Replace '...' with <Label>...</Label>
+  return reactStringReplace(message, /'([^']+)'/gim, (match, i) => (
```

Comment:
I removed previously `message.replace(/"/g, "'")` I think we don't need to replace `"`